### PR TITLE
[Task 1862940] new creation UI for storage account

### PR DESCRIPTION
**Pull request by shenqianjin**
[AB#1862940](https://dev.azure.com/mseng/a4d27ce2-a42d-4b71-8eef-78cee9a9728e/_workitems/edit/1862940)

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/task/CreateStorageAccountTask.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    public CreateStorageAccountTask(StorageAccountConfig config) {
+        this.config = config;
+    }
+
+    public StorageAccount execute() {
+        final Operation operation = TelemetryManager.createOperation(ActionConstants.StorageAccount.CREATE);
+        try {
+            operation.start();
+            final String subscriptionId = config.getSubscription().getId();
+            EventUtil.logEvent(EventType.info, operation, Collections.singletonMap(TelemetryConstants.SUBSCRIPTIONID, subscriptionId));
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/task/CreateStorageAccountTask.java]: avoid using ventUtil.logEvent

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/CreateStorageAccountAction.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    }
+
+    @Override
+    public void actionPerformed(NodeActionEvent e) {
+        final Project project = (Project) module.getProject();
+        AzureSignInAction.requireSignedIn(project, () -> doActionPerformed(true, project));
+    }
+
+    private void doActionPerformed(boolean isLoggedIn, Project project) {
+        try {
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/CreateStorageAccountAction.java]: delete from line 50 to 59

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/task/CreateStorageAccountTask.java (hunk 1 above)
Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/task/CreateStorageAccountTask.java]: All resources used this code. let us consider to refactor in a seperate task.

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/CreateStorageAccountAction.java (hunk 2 above)
Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/CreateStorageAccountAction.java]: updated.

**Review by andxu — DISMISSED** [review]

**Review by andxu — DISMISSED** [review]
LGTM

**Pull request merged**